app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java

```diff
@@ -293,17 +295,36 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
     }
 
     private void importData(String path) {
-        if (permission) {
-            if (StatisticsFragment.parsedData == null || parsedData == null) {
-                progressDialog.show();
-                ParseDataAsync parseDataAsync = new ParseDataAsync(path);
-                parseDataAsync.execute();
-            }
+        if (permission && (StatisticsFragment.parsedData == null || parsedData == null)) {
+            toggleMenuItem(menu, !isPlaying);
+            progressDialog.show();
+            ParseDataAsync parseDataAsync = new ParseDataAsync(path);
+            parseDataAsync.execute();
         } else {
             getRuntimePermissions();
         }
     }
 
+    /**
+     * Toggle menu items.
+     *
+     * @param menu
+     * @param isPlaying
+     */
+    private void toggleMenuItem(Menu menu, boolean isPlaying) {
+        MenuItem play = menu.findItem(R.id.play_graph);
+        play.setVisible(isPlaying);
+        MenuItem stop = menu.findItem(R.id.stop_data);
+        stop.setVisible(!isPlaying);
```

Comment:
I dunno how to explain more. We have to be logical in programming.

Take this as an example. "**Stop icon should be visible if it is playing something**" right? What line 318 implies is otherwise. Use variable naming and their context in a proper sense. I just gave you an "example". Don't just copy paste it without understanding where and how to put it to use.